hotsos/core/plugins/kubernetes.py

```diff
@@ -110,10 +113,15 @@ def __init__(self, *args, **kwargs):
                                        other_snaps=snap_deps)
         self.pebble = PebbleHelper(service_exprs=SERVICES)
         self.systemd = SystemdHelper(service_exprs=SERVICES)
+        self.microk8s = SnapPackageHelper(core_snaps=MICROK8S_SNAP)
+        if self.microk8s.core:
+            microk8s_deps = snap_deps + MICROK8S_SNAP
+            self.snaps = SnapPackageHelper(core_snaps=K8S_PACKAGES,
+                                           other_snaps=microk8s_deps)
 
     @property
     def plugin_runnable(self):
-        if self.apt.core or self.snaps.core:
+        if self.apt.core or self.snaps.core or self.microk8s.core:
```

Comment:
is there a benefit to treating the microk8s snap separately to the other snaps? this will result in two lookups and if it has not intended benefit i would prefer we included microk8s with the other snaps.

Reply:
My only reasoning was that in the future if we want to separate checks between microk8s / k8s we would have a way to have it done.
I agree it becomes cleaner/easier to just add the microk8s with the other snaps, and making the code change in the future when necessary is also easy, I will fix this.

Reply:
@dosaboy fixed on https://github.com/canonical/hotsos/commit/9dcfaef97f9a1735242377ffc426e091986d47b1